runtime/src/bank.rs

```diff
@@ -4242,15 +4246,18 @@ impl Bank {
             "commit_transactions() working on a bank that is already frozen or is undergoing freezing!"
         );
 
-        self.increment_transaction_count(tx_count);
+        self.increment_transaction_count(executed_transactions_count);
         self.increment_signature_count(signature_count);
 
-        inc_new_counter_info!("bank-process_transactions-txs", tx_count as usize);
+        inc_new_counter_info!(
+            "bank-process_transactions-txs",
+            executed_transactions_count as usize
+        );
         inc_new_counter_info!("bank-process_transactions-sigs", signature_count as usize);
 
         if !sanitized_txs.is_empty() {
             let processed_tx_count = sanitized_txs.len() as u64;
-            let failed_tx_count = processed_tx_count.saturating_sub(tx_count);
+            let failed_tx_count = processed_tx_count.saturating_sub(executed_transactions_count);
```

Comment:
Note that this and the counters below look wrong before and after your changes because `processed_tx_count` is actually "attempted execution count".

Reply:
Yeah so that means this `failed_tx_count` includes transactions that were not even executed?

Reply:
Yeah, that's my understanding, worth fixing

Reply:
Fixed up here: https://github.com/solana-labs/solana/pull/22802/commits/c1c2a6509ef4ab7b06c3b5b940de8d9e265fd2b8